client/src/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './App.css';
+import RegisterPage from './Components/RegisterPage';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <RegisterPage />
+      </div>
+    );
+  }
+}
+
```

Comment:
Good job working on the stretch, setting the register page component. You should also work on the login, so that the logged user can see the users in the database. 
Overall great job.